.github/workflows/ci-integration-tests.yml

```diff
@@ -28,7 +28,15 @@ jobs:
       - name: Start services
         run: docker-compose -f tests/docker/docker-compose.yml up -d && sleep 60
       - name: Test
-        run: sbt -Dsbt.color=always -Dsbt.supershell=false "project tests" test
+        run: |
```

Comment:
It may be simpler to do the check on curl here ?

Reply:
I suppose it would be more self-contained if it were here 🤔  I will have a look at it later